index.bs

```diff
@@ -1394,21 +1404,38 @@ To <dfn>parse filter data</dfn> given a |value|:
 Issue: Determine whether to limit [=string/length=] or
 [=string/code point length=] for |filter| and |d| above.
 
+To <dfn>parse filter config</dfn> given a |value|:
+
+1. If |value| is not a [=map=], return null.
+1. Let |lookbackWindow| be null.
+1. If |value|["`lookback_window`"] [=map/exists=]:
+    1. If |value|["`lookback_window`"] is not a non-negative integer, return null.
```

Comment:
Is there a reason to allow 0 here? Should we require the integer to be positive instead? As written, this is incompatible with the `filter config` definition, which says the duration is positive.